muterCore/version.swift

```diff
@@ -0,0 +1 @@
+public let version = "v13"
```

Comment:
can this be updated as part of ShipIt.sh?

Reply:
it is!

https://github.com/muter-mutation-testing/muter/blob/17f66435a445c27dfcc89d81a7df10d5cf2eb6ee/Scripts/shipIt.sh#L8

Reply:
#176 😉
![](https://i.imgflip.com/4g2nzv.jpg)

Reply:
hahahahahahahahaha I read it wrong, sorry!
I thought you where talking about the version value being exposed...yes, I'm crazy :P